rest/changes_api_test.go

```diff
@@ -181,6 +181,159 @@ func TestDocDeletionFromChannel(t *testing.T) {
 	assert.Equals(t, response.Code, 200)
 }
 
+
```

Comment:
Adding a copy of these structs to the rest package doesn't feel like the right way to do this - it feels like a hack, and the definitions are likely to diverge.

Can this test be run in db (maybe following the pattern in changes_test.go)?

If not, and this absolutely needs to be here, I think we're better off making SyncData and RevTreeList public in db.